README.md

```diff
@@ -59,6 +59,14 @@ Or install it yourself as:
 $ gem install turbo_tests
 ```
 
+## Setup
+
+Create test databases
+
+```bash
+$ bundle exec turbo_tests --create
+```
```

Comment:
### Create test databases before running tests

1. Start `TEST_ENV_NUMBER` from `1`

```bash
export PARALLEL_TEST_FIRST_IS_1=true
```

2. Create databases

```bash
bundle exec rake db:create RAILS_ENV=test
```

Reply:
That doesn't create the necessary test databases. That will just create a single "_test" database, not one database per cpu.

Here is your suggestion:
```
❯ export PARALLEL_TEST_FIRST_IS_1=true
❯ bundle exec rake db:create RAILS_ENV=test
Database 'myapp_test' already exists
```

Here is my suggestion with this PR:
```
❯ bundle exec turbo_tests --create
Database 'myapp_test5' already exists
Database 'myapp_test9' already exists
Database 'myapp_test10' already exists
Database 'myapp_test1' already exists
Database 'myapp_test3' already exists
Database 'myapp_test7' already exists
Database 'myapp_test8' already exists
Database 'myapp_test6' already exists
Database 'myapp_test2' already exists
Database 'myapp_test4' already exists
```

The way to do this without any changes to turbo_tests is first install the parallel_test gem as a direct dependency, and then run this:
```
❯ export PARALLEL_TEST_FIRST_IS_1=true
❯ bundle exec rake parallel:create
Database 'myapp_test3' already exists
Database 'myapp_test1' already exists
Database 'myapp_test9' already exists
Database 'myapp_test2' already exists
Database 'myapp_test8' already exists
Database 'myapp_test5' already exists
Database 'myapp_test6' already exists
Database 'myapp_test10' already exists
Database 'myapp_test4' already exists
Database 'myapp_test7' already exists
```
But the problem with this is it it requires having the parallel_tests gem installed as a direct dependency. And this is a direct dependency that is only used for the purpose of this one-time command. With my PR, the user is no longer required to install parallel_tests as a direct dependency. They can just install turbo_tests which acts as their gateway to parallel_tests.